electionleaflets/settings/testing.py

```diff
@@ -1,11 +1,16 @@
 import os
+from tempfile import mktemp
 
 from .base import *  # noqa: F403
+os.environ["DJANGO_ALLOW_ASYNC_UNSAFE"] = "true"
 
-DEFAULT_FILE_STORAGE = "django.core.files.storage.FileSystemStorage"
-STATICFILES_STORAGE = "django.core.files.storage.FileSystemStorage"
-TEST_RUNNER = "django.test.runner.DiscoverRunner"
+DEFAULT_FILE_STORAGE = "electionleaflets.storages.TempUploadLocalMediaStorage"
+STATICFILES_STORAGE = "electionleaflets.storages.StaticStorage"
 
-MEDIA_ROOT = root("test_media",)  # noqa: F405
+MEDIA_ROOT = mktemp()
 
 INSTALLED_APPS.append("aloe_django")  # noqa: F405
```

Comment:
Are we actually still using aloe?
As far as I can see, none of the code in
https://github.com/DemocracyClub/electionleaflets/tree/master/electionleaflets/apps/leaflets/features
is being discovered or executed by pytest.
Is the intention that it should be, or should this whole lot be binned?

Reply:
Punted to https://github.com/DemocracyClub/electionleaflets/issues/274